modules/viewport-mercator-project/src/web-mercator-utils.js

```diff
@@ -229,7 +238,7 @@ export function getProjectionParameters({
   const farZ = Math.cos(Math.PI / 2 - pitchRadians) * topHalfSurfaceDistance + altitude;
 
   return {
-    fov: 2 * Math.atan(height / 2 / altitude),
+    fov: 2 * halfFov,
     aspect: width / height,
     focalDistance: altitude,
     near: nearZMultiplier,
```

Comment:
near/far will have to be updated to take scale into account, since z depends on scale now.

Reply:
z has always been dependent on scale. Prior to this PR the scale is applied in common space. After it is applied in the view matrix.

z scaling is not changed in this PR, only x and y, so that we can use a uniform fov.